homeassistant/components/climate/__init__.py

```diff
@@ -377,6 +377,14 @@ def _report_turn_on_off(feature: str, method: str) -> None:
             # Return if integration has migrated already
             return
 
+        supported_features = self.supported_features
+        if supported_features & (
+            ClimateEntityFeature.TURN_ON | ClimateEntityFeature.TURN_ON
```

Comment:
Was this meant to be on/on or on/off?

Reply:
Great catch. Will update it!

Reply:
https://github.com/home-assistant/core/pull/121398